### DISCUSSION TITLE
README.MD appears instead of my page!

### DISCUSSION THREAD

**Discussion by anthonyArellano1923** [community/community#126252] [Other Feature Feedback, Questions, & Ideas] [answered]
### GitHub Pages

Question

### Body

Hi there! I'm taking some github lessons and i was requested to publish my page using GitHub Pages but after following the instructions given by GitHub, the page published the content of the README.md file instead of the page i created. I went to Chromes inspector and the html file was there, but my page just doesn't appears! Does anyone knows how to fix that?

**Comment by davevad93** [comment, marked as answer]
Hi @anthonyArellano1923 

Make sure that the main `html` file is in the root of your repo and that is named `index.html`.
